api/oci/extensions/repositories/ocireg/namespace.go

```diff
@@ -159,18 +151,18 @@ func (n *NamespaceContainer) GetArtifact(i support.NamespaceAccessImpl, vers str
 	}
 	_, acc, err := blobData.GetBlobData(desc.Digest)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("get blob data failed for digest %s: %w", desc.Digest, err)
```

Comment:
here you know, that a manifest is fetched, this should be stated in the message if you create a detailed message.